connect_ext_test.go

```diff
@@ -2423,7 +2423,9 @@ func TestClientDisconnect(t *testing.T) {
 			assert.NotNil(t, err)
 			<-gotResponse
 			assert.NotNil(t, handlerReceiveErr)
-			assert.Equal(t, connect.CodeOf(handlerReceiveErr), connect.CodeCanceled)
+			if !assert.Equal(t, connect.CodeOf(handlerReceiveErr), connect.CodeCanceled) {
+				t.Logf("handlerReceiveErr: %v", handlerReceiveErr)
```

Comment:
Unrelated, but noticed in testing. Raising the log to help debug better if seen in CI.